app/assets/stylesheets/components/message.scss

```
@@ -172,8 +172,8 @@
 
 .folder-heading {
 
-  .grid-row & {
-    margin: $gutter-half 0 20px 0;
+  .govuk-grid-row & {
+    margin: govuk-spacing(3) 0 20px 0;
```

`20px` is the same as `$gutter-two-thirds`
```suggestion
    margin: govuk-spacing(3) 0 govuk-spacing(4) 0;
```